integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -574,11 +574,34 @@ export class CdcJsService implements OnDestroy {
           if (currentEmail !== userDetails.uid) {
             this.logoutUser();
           }
+          this.handleProfileUpdateTostMessage(response);
         });
       });
     }
   }
 
+  /**
+   * handle toast message on profile update
+   * @param response
+   *
+   */
+  handleProfileUpdateTostMessage(response?: any) {
```

Comment:
make the method from handleProfileUpdateTostMessage - > handleProfileUpdateToastMessage

Reply:
Done